gitea/releases.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package gitea
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+// Release represents a repository release
+type Release struct {
+	ID           int64     `json:"id"`
+	TagName      string    `json:"string"`
```

Comment:
this should be `json:"tag_name"` according to https://developer.github.com/v3/repos/releases/